src/lqcut.jl

```diff
@@ -127,23 +118,23 @@ function lq_drift_time_correction(
 
     return result, report
 end
-export lq_drift_time_correction
+export lq_ctc_correction
 
 """
-    DEP_µ::Unitful.Energy, DEP_σ::Unitful.Energy, e_cal::Vector{<:Unitful.Energy}, lq_classifier::Vector{Float64}; cut_sigma::Float64=3.0, truncation_sigma::Float64=2.0)
+    DEP_µ::Unitful.Energy, DEP_σ::Unitful.Energy, e_cal::Vector{<:Unitful.Energy}, lq_classifier::Vector{<:AbstractFloat}; cut_sigma::Float64=3.0, dep_sideband_sigma::Float64=4.5, cut_truncation_sigma::Float64=2.0)
```

Comment:
Again here: the docstring is missing the name of the function: `lq_cut` and the description of the arguments (see #80)